### PR TITLE
[TTS][Qwen3] Reduce first audio latency

### PR DESCRIPTION
**Pull request by luojiaxuan**
Follow-up to #1286. Reduces Qwen3-TTS first-audio latency by:

- batching reference-code extraction and overlapping it with speaker embedding
- using CUDA graphs for fixed-size initial vocoder decode
- prioritizing follow-up decode without delaying first audio

H100 Seed-TTS EN full set, matched config and hardware, 1088 requests per setup.

| Concurrency | Current TTFA p95 | Baseline | Current C200 | Baseline C200 | Current/Baseline QPS | Current/Baseline EN WER |
|---:|---:|---:|---:|---:|---:|---:|
| 8 | 0.266s | 0.330s | 84.74% | 0% | 6.114 / 2.921 | 1.06% / 1.07% |
| 16 | 0.320s | 0.497s | 12.41% | 0% | 7.746 / 3.107 | 1.06% / 1.06% |
| 32 | 0.541s | 0.895s | 3.03% | 0% | 9.612 / 3.155 | 1.05% / 1.06% |

All runs completed 1088/1088. First payload p95 is 3.75 KiB. This change does not modify talker sampling.

Validation:
- repository pre-commit passed
- reference encoder unit tests: 14 passed
- Python compileall passed
- full Qwen3-TTS coverage runs in CI

### CODE REVIEW

**Comment by luojiaxuan** [comment]
@SandyLuXY #1286 has landed. This is the isolated TTFA follow-up; please approve for merge.

File: sglang_omni/models/qwen3_tts/request_builders.py [hunk 1]
```diff
@@ -617,18 +737,38 @@ def options_key(self, item: _Qwen3TTSAdhocReferenceInput) -> str:
...
         with torch.no_grad():
-            prompt_items = self._wrapper.create_voice_clone_prompt(
-                ref_audio=item.ref_audio,
-                ref_text=item.ref_text,
-                x_vector_only_mode=item.x_vector_only_mode,
+            normalized = self._wrapper._normalize_audio_inputs([item.ref_audio])
+            if len(normalized) != 1:
+                raise ValueError("Qwen3-TTS expects exactly one reference audio")
+            waveform, sample_rate = normalized[0]
+            ref_code_future = self._ref_code_batcher.submit(waveform, sample_rate)
```
Comment (SandyLuXY) [inline sglang_omni/models/qwen3_tts/request_builders.py:801]: The production preprocessor is serial (`max_concurrency=1`, no batch function), and this hook waits on the future before it can dequeue another request, so the queue never forms a cross-request batch; a two-request production repro recorded tokenizer batches `[1, 1]`. Please correct me if I'm wrong 🤔️

File: sglang_omni/models/qwen3_tts/request_builders.py [hunk 2]
```diff
@@ -617,18 +737,38 @@ def options_key(self, item: _Qwen3TTSAdhocReferenceInput) -> str:
...
+            speaker_sample_rate = self._model.speaker_encoder_sample_rate
+            if sample_rate != speaker_sample_rate:
+                import librosa
+
+                speaker_waveform = librosa.resample(
+                    y=speaker_waveform.astype("float32"),
+                    orig_sr=int(sample_rate),
+                    target_sr=speaker_sample_rate,
+                )
+            speaker_embedding = self._model.extract_speaker_embedding(
```
Comment (SandyLuXY) [inline sglang_omni/models/qwen3_tts/request_builders.py:812]: This worker and `extract_speaker_embedding()` both use CUDA stream 0 because neither path selects a stream, so their GPU kernels serialize. Shall we run reference-code encoding on a dedicated stream, hand the result off with an event/`wait_stream` before resolving the future, and cover the real CUDA path?

File: sglang_omni/models/qwen3_tts/request_builders.py (hunk 1 above)
Comment (luojiaxuan) [reply sglang_omni/models/qwen3_tts/request_builders.py:801]: Good catch, you are right — the preprocessor was serial so the batch never really formed. Fixed in 83da919: preprocessing now runs with max_concurrency=8 (same as batcher max_batch_size), so cross-request batch can actually form. Also added a two-request test through the production preprocess path, tokenizer batch is [2] now.

File: sglang_omni/models/qwen3_tts/request_builders.py (hunk 2 above)
Comment (luojiaxuan) [reply sglang_omni/models/qwen3_tts/request_builders.py:812]: Done in 83da919. Ref-code encoding now runs on a dedicated CUDA stream (picked from model.device), and the worker records an event on it and waits before resolving the future, so the default stream is not touched anymore — speaker embedding can really overlap. Consumer side does record_stream so the allocator will not recycle the side-stream block too early. Also added a real CUDA unit test to cover this path (asserts encode runs on the non-default stream).

**Review by SandyLuXY — APPROVED** [review]
lgtm
